packages/usa-input-mask/src/content/usa-input-mask-phone.json

```diff
@@ -1,9 +1,9 @@
 {
-  "label": "US Telephone Number",
   "hint": "For example, 123-456-7890",
   "id": "tel",
-  "type": "tel",
-  "inputmode": "numeric",
-  "placeholder": "___-___-____",
-  "pattern": "\\d{3}-\\d{3}-\\d{4}"
+  "inputmode": "tel",
+  "label": "U.S. Telephone Number",
+  "mask": "(999) 999-9999",
+  "placeholder": "(___) ___-____",
+  "type": "tel"
```

Comment:
I tried experimenting with changing the input `type` and opening the keyboard on mobile but nothing changed.

I'm not sure if there is another assistive technology that this is utilized with but wanted to flag it!